templates/config.j2

```diff
@@ -13,3 +14,4 @@ CONTACT_EMAIL={{ __letsencrypt.contact }}
 OWNER={{ __letsencrypt.owner }}
 SERVICES=( {{ __letsencrypt.restart_services | join(' ') }} )
 WELLKNOWN={{ __letsencrypt.well_known }}
+PREFERRED_CHAIN={{ __letsencrypt.preferred_chain }}
```

Comment:
Do it like this instead:

```
case $CA in
    $prod_ca) PREFERRED_CHAIN='ISRG Root X1' ;;
    $staging_ca) PREFERRED_CHAIN='(STAGING) Pretend Pear X1' ;;
    *) PREFERRED_CHAIN={{ __letsencrypt.preferred_chain|default('') }} ;;
esac
```

Reply:
Done in https://github.com/TritonDataCenter/ansible-triton-dehydrated/commit/63a905a0011763a0c5740bba7d7016ebd69a11f3.